src/runmode-af-packet.c

```diff
@@ -216,11 +216,40 @@ void *ParseAFPConfig(const char *iface)
         }
     }
 
-    (void)ConfGetChildValueBoolWithDefault(if_root, if_default, "use-mmap", (int *)&boolval);
+    if (ConfGetChildValueBoolWithDefault(if_root, if_default, "use-mmap", (int *)&boolval) == 1) {
+        if (boolval) {
+            aconf->flags |= AFP_RING_MODE;
+        } else {
+            SCLogInfo("Disabling mmaped capture on iface %s",
+                    aconf->iface);
+        }
+    } else {
+        aconf->flags |= AFP_RING_MODE;
+    }
+
+    (void)ConfGetChildValueBoolWithDefault(if_root, if_default, "mmap-locked", (int *)&boolval);
```

Comment:
On 18-04-16 14:19, Eric Leblond wrote:
> Indeed, you made a good comment for l.232 ;) I will fix that.
> 

I'm getting fuzzy from all the fuzzing I guess ;)